en/streaming-integrator/docs/ref/performance-analysis-results.md

```diff
@@ -1,7 +1,34 @@
 # Performance Analysis Results
 
 !!!note
-    These performance statistics were taken when the load average was below 3.8 in the 4 core instance.
+    - These performance statistics were taken when the load average was below 3.8 in the 4 core instance.<br/><br/>
+
+    
+## Performance analysis results summary
+
+The recommended CPU and memory specifications for Docker containers are as follows:
+
+- **CPU**: 4 cores
+
+- **Memory**: 8GB
+
+- **Xms**: 2GB
+
+- **Xmx**: 4GB
+
+The exact specifications used in the use cases listed in this section are summarised in the table below:
+
+| **Scenario**                                          | **CPU**   | **Memory** | **Input TPS** | **Input Message Size** | **Total Message Count** | **Output TPS** |
+|-------------------------------------------------------|-----------|------------|---------------|------------------------|-------------------------|----------------|
+| [Consuming events using Kafka source](#consuming-events-using-kafka-source)| | | 180K | | | 1.1M |
```

Comment:
Use the https://aws.amazon.com/ec2/instance-types/#Compute_Optimized 's c5 column to check the CPU and Memory details that was used

Reply:
Added the CPU and memory details from the link provided. Thank you.